coalib/bearlib/aspects/decorators.py

```diff
@@ -0,0 +1,37 @@
+
+from functools import wraps
+
+from .taste import Taste
+from .meta import aspectclass
+
+
+def map_setting_to_aspect(**aspectable_setting):
+    """
+    Override function arguments value with coala's aspect and taste.
+
+    This decorator can be used by ``Bear.run()`` to automatically map and
+    override bear's setting value with their equivalent aspect or taste.
+
+    :param aspectable_setting:
+        A dictionary of settings as keys and their equivalent aspect or taste
+        as value.
+    """
+    def _func_decorator(func):
+        @wraps(func)
+        def _new_func(self, *args, **kwargs):
+            if hasattr(self, 'section') and self.section.aspects:
```

Comment:
Why `hasattr(self, 'section')` ; i think it should raise an exception if there is no `.section`?

Reply:
Yeah right `hasattr(self, 'section')` is not necessary here.